XSLT/utkMODStoMODS.xsl

```diff
@@ -1,282 +1,163 @@
+            artifacts)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300027275">corporation reports</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300026877">correspondence</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300015387">diagrams</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300033973">drawings (visual works)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300028881">ephemera (general)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300028048">filmstrips</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300015578">illustrations</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300028569">manuscripts (document
+            genre)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300028094">maps (documents)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300027440">minutes (administrative
+            records)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300037316">money</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300263857">moving images</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300127173">negatives (photographic)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300311889">objects</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300202595">oral histories (document
+            genres)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300033618">paintings (visual works)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300220572">pamphlets</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300026657">periodicals</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300046300">photographs</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300026816">postcards</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300027221">posters</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300264821">printed ephemera</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300041273">prints (visual works)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300247936">record covers</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300026685">records (documents)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300026427">scores</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300123013">signs (declatory or advertising
+            artifacts)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300028633">sound recordings</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300127478">trasnparencies</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300028682">video recordings</f>
+    </xsl:param>
+
+    <!-- Drop Unknown Names -->
+    <xsl:template match="name[namePart[matches(text(), lower-case('unknown'))]]"/>
```

Comment:
I think there could be a logic issue here - sorry for catching this so late: I think we want `name[namePart[matches(lower-case(text()), 'unknown'))]]`.